### PR TITLE
Added information and news about the official PPA

### PR DESCRIPTION
**Pull request by Photon89**

### CODE REVIEW

File: _posts/2021-08-10-ppa_news.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title: Official PPA revived
+author: Michael Kogan
+permalink: /ppa-revival
+tags: [ppa]
```
Comment (DarthGandalf) [inline _posts/2021-08-10-ppa_news.md]: news?

Comment (Photon89) [reply _posts/2021-08-10-ppa_news.md]: Yeah, did I mess up something?

Comment (DarthGandalf) [reply _posts/2021-08-10-ppa_news.md]: The "news" tag, it's not here

File: _posts/2021-08-10-ppa_news.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+layout: post
+title: Official PPA revived
+author: Michael Kogan
+permalink: /ppa-revival
```
Comment (DarthGandalf) [inline _posts/2021-08-10-ppa_news.md]: Why do you need the permalink for it? The default one would contain the date, which is more appropriate for news item, I think

Comment (Photon89) [reply _posts/2021-08-10-ppa_news.md]: Right, updated tags and removed permalink!

**Pull request merged**